fortinet-fortiauthenticator/info.json

```diff
@@ -0,0 +1,889 @@
+          "visible": true,
+          "value": ""
+        },
+        {
+          "title": "Email",
+          "type": "text",
+          "name": "email",
+          "description": "Email address for the user",
+          "tooltip": "Must be a valid e-mail address",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "value": ""
+        },
+        {
+          "title": "First Name",
+          "type": "text",
+          "name": "first_name",
+          "description": "First name of the user",
+          "tooltip": "max length = 30",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "value": ""
+        },
+        {
+          "title": "Last Name",
+          "type": "text",
+          "name": "last_name",
+          "description": "Last name of the user",
+          "tooltip": "max length = 30",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "value": ""
+        },
+        {
+          "title": "Account Status",
+          "type": "checkbox",
+          "name": "active",
+          "description": "Account Status",
+          "tooltip": "Status of account",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "value": ""
+        }
+      ]
+    },
+    {
+      "operation": "update_user_status",
+      "title": "Update Local User Status",
+      "description": "Activate/Deactivates local user.",
+      "enabled": true,
+      "category": "investigation",
+      "annotation": "update_user_status",
+      "output_schema": {
+        "active": "",
+        "address": "",
+        "change_password": "",
+        "city": "",
+        "country": "",
+        "custom1": "",
+        "custom2": "",
+        "custom3": "",
+        "email": "",
+        "expires_at": "",
+        "fido": "",
+        "first_name": "",
+        "ftk_only": "",
+        "ftm_act_method": "",
+        "id": "",
+        "last_name": "",
+        "mail_host": "",
+        "mail_routing_address": "",
+        "mobile_number": "",
+        "phone_number": "",
+        "reason": "",
+        "recovery_by_question": "",
+        "resource_uri": "",
+        "state": "",
+        "token_auth": "",
+        "token_fas": "",
+        "token_serial": "",
+        "token_type": "",
+        "user_groups": [],
+        "username": ""
+      },
+      "parameters": [
+        {
+          "title": "UserID",
+          "type": "text",
+          "name": "userid",
+          "description": "UserID to deactivate.",
+          "tooltip": "UserID to deactivate. ",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "value": ""
+        },
+        {
+          "title": "Active/Deactive",
+          "type": "checkbox",
+          "name": "active",
+          "description": "UserID to deactivate.",
+          "tooltip": "UserID to deactivate. ",
```

Comment:
Please specify how this param going to work like Enable (true) to activate the User or Disable(False) to deactivate the user.